2025-07-03 21:27:50  [5] ce80606054: ‚ùå ColabFold failed
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
E0000 00:00:1751549098.964457 2474366 cuda_dnn.cc:8310] Unable to register cuDNN factory: Attempting to register factory for plugin cuDNN when one has already been registered
E0000 00:00:1751549098.969077 2474366 cuda_blas.cc:1418] Unable to register cuBLAS factory: Attempting to register factory for plugin cuBLAS when one has already been registered

WARNING: You are welcome to use the default MSA server, however keep in mind that it's a
limited shared resource only capable of processing a few thousand MSAs per day. Please
submit jobs only from a single IP address. We reserve the right to limit access to the
server case-by-case when usage exceeds fair use. If you require more MSAs: You can 
precompute all MSAs with `colabfold_search` or host your own API and pass it to `--host-url`


  0%|          | 0/150 [elapsed: 00:00 remaining: ?]
SUBMIT:   0%|          | 0/150 [elapsed: 00:00 remaining: ?]
COMPLETE:   0%|          | 0/150 [elapsed: 00:00 remaining: ?]
COMPLETE: 100%|‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà| 150/150 [elapsed: 00:00 remaining: 00:00]
COMPLETE: 100%|‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà‚ñà| 150/150 [elapsed: 00:05 remaining: 00:00]
WARNING: All log messages before absl::InitializeLog() is called are written to STDERR
I0000 00:00:1751549112.809399 2474366 mlir_graph_optimization_pass.cc:401] MLIR V1 optimization pass is not enabled
Traceback (most recent call last):
  File "/gpfs/spack/opt/linux-rocky8-icelake/gcc-8.5.0/anaconda3-2022.10-4dp3trddxrrzcg6rozuot7ckgh3zjche/envs/localcolabfold/bin/colabfold_batch", line 10, in <module>
    sys.exit(main())
  File "/gpfs/spack/opt/linux-rocky8-icelake/gcc-8.5.0/anaconda3-2022.10-4dp3trddxrrzcg6rozuot7ckgh3zjche/envs/localcolabfold/lib/python3.10/site-packages/colabfold/batch.py", line 2033, in main
    run(
  File "/gpfs/spack/opt/linux-rocky8-icelake/gcc-8.5.0/anaconda3-2022.10-4dp3trddxrrzcg6rozuot7ckgh3zjche/envs/localcolabfold/lib/python3.10/site-packages/colabfold/batch.py", line 1571, in run
    results = predict_structure(
  File "/gpfs/spack/opt/linux-rocky8-icelake/gcc-8.5.0/anaconda3-2022.10-4dp3trddxrrzcg6rozuot7ckgh3zjche/envs/localcolabfold/lib/python3.10/site-packages/colabfold/batch.py", line 466, in predict_structure
    files.get("unrelaxed","pdb").write_text(protein_lines)
  File "/gpfs/spack/opt/linux-rocky8-icelake/gcc-8.5.0/anaconda3-2022.10-4dp3trddxrrzcg6rozuot7ckgh3zjche/envs/localcolabfold/lib/python3.10/pathlib.py", line 1154, in write_text
    with self.open(mode='w', encoding=encoding, errors=errors, newline=newline) as f:
  File "/gpfs/spack/opt/linux-rocky8-icelake/gcc-8.5.0/anaconda3-2022.10-4dp3trddxrrzcg6rozuot7ckgh3zjche/envs/localcolabfold/lib/python3.10/pathlib.py", line 1119, in open
    return self._accessor.open(self, mode, buffering, encoding, errors,
FileNotFoundError: [Errno 2] No such file or directory: '/gpfs/work/bio/shuleihe23/alpha_fold/run_colabfold_pipeline/output/pdb/ce80606054/ce80606054_unrelaxed_alphafold2_ptm_model_1_seed_000.pdb'
2025-07-03 21:27:50  üéâ Completed 6 entries.
